api/nodes/serializers.py

```diff
@@ -145,6 +146,7 @@ class NodeCitationSerializer(JSONAPISerializer):
     publisher = ser.CharField(allow_blank=True, read_only=True)
     type = ser.CharField(allow_blank=True, read_only=True)
     doi = ser.CharField(allow_blank=True, read_only=True)
+    custom_citation_text = ser.CharField(allow_blank=True, required=False)
```

Comment:
I'm not sure this change is necessary or desired, especially since the custom citation can be updated by making a request to the nodes endpoint.

This results in a serialization that's a bit confusing as well since it's partially a custom citation but partially returning other citation details:
```
{
    ...
    "attributes": {
		"title": "<node_title>",
		"author": { <contrib_list> },
		"publisher": "OSF",
		"type": "webpage",
		"doi": "10.1234",
		"custom_citation_text": ""
	}
}
```

I would think it's fine to leave this view and serializer as-is and only implement the custom citation parts on the node serializer (since it's really just another node field that happens to have the word citation in it).